Update ftw-baselines readme (#104)
fieldsoftheworld/ftw-baselines: README.md

This repository provides the codebase for working with the [FTW dataset](https://beta.source.coop/repositories/kerner-lab/fields-of-the-world/description/), including tools for data pre-processing, model training, and evaluation.

> ## ⚠️ Note on Repository Structure <!-- omit in toc -->
> 
> The Fields of The World Command Line Inferface (FTW CLI),`ftw-tools`, currently lives in this `ftw-baselines` repository due to legacy reasons. We plan to migrate the FTW CLI and related tools into an `ftw-tools` repository soon. Until then, the latest and most complete version of the FTW CLI still lives in `ftw-baselines`.
> 

Reviewer: We could use these GitHub blocks to format them nicely.

```suggestion
> [!NOTE]  
> The Fields of The World Command Line Inferface (FTW CLI),`ftw-tools`, currently lives in this `ftw-baselines` repository due to legacy reasons. We plan to migrate the FTW CLI and related tools into an `ftw-tools` repository soon. Until then, the latest and most complete version of the FTW CLI still lives in `ftw-baselines`.

```

### Verify PyTorch installation and CUDA availability

If you are using GPU, verify that PyTorch and CUDA are installed correctly:

```bash
python -c "import torch; print(torch.cuda.is_available())"
```

### Setup FTW CLI

> To create the `ftw` command-line tool, run the following:

Reviewer: ```suggestion
To install the `ftw` command-line tool on your computer, run the following:
```

```bash
pip install ftw-tools
```

> If you plan to develop on the FTW CLI at all, you will run one of the following commands from your cloned repository:

Reviewer: ```suggestion
### Development

If you plan to make changes to the FTW CLI at all, you will run one of the following commands from within your cloned repository:

```

                        countries.
  --no-unpack           If set, the script will NOT unpack the downloaded
                        files.
  --help                Show this message and exit.
```

If you had `--no-unpack` enabled during download, you can manually unpack the downloaded files using the `unpack` command.
This will create a `ftw` folder under the given folder after unpacking.

```text
Usage: ftw data unpack [OPTIONS] [INPUT]

  Unpack the downloaded FTW dataset. Specify the folder where the data is
  located via INPUT. Defaults to './data'.

Options:
  --help  Show this message and exit.
```

### Download the FTW Baseline Dataset

To download and unpack the complete FTW Baseline Dataset, use following command:

```bash
ftw data download
```

To download and unpack the specific country or set of countries, use following command:

```bash
ftw data download --countries belgium,kenya,vietnam
```

*Note:* Make sure to avoid adding any space in between the list of comma seperated countries.

### Visualize the FTW Baseline Dataset

Explore `visualize_dataset.ipynb` to know more about the dataset.

![Sample 1](/assets/sample1.png)
![Sample 2](/assets/sample2.png)

> ## CC-BY (or equivalent) trained models

Reviewer: ```suggestion
## CC-BY vs. the full model
```
